pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -98,6 +98,7 @@ func (r *ControlPlaneReconciler) Reconcile() (reconcile.Result, error) {
 	// these components have to be installed in the specified order
 	orderedComponents := []string{
 		"istio", // core istio resources
+		"istio/charts/istio_cni",
```

Comment:
does this require any cert injection from citadel?

Reply:
no.